vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/PgEncoder.java

```diff
@@ -179,6 +181,20 @@ void writeClosePortal(String portal) {
     out.setInt(pos + 1, out.writerIndex() - pos - 1);
   }
 
+  void writeClosePreparedStatement(long statementName) {
```

Comment:
I don't see this code called in the PR, am I missing something ?

Reply:
This is used for closing Postgres named prepared statements, we could add support for that in another PR as stated in the proposal.

It's not useful in this PR, I will remove this.

Reply:
yes I remember this part and in reality it should be called when one is calling `PreparedStatement.close()` method.

can we have this case tested and implemented with `io.vertx.pgclient.impl.codec.CloseStatementCommandCodec` ? I think it used to work in the former Reactive PG Client.

Reply:
in other words calling `PreparedStatement#close()` when caching is disabled should actually release this resource in PostgreSQL and this code should be called.